Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView/ConsentTransparencyView.swift

```diff
@@ -148,7 +153,7 @@ private extension ConsentTransparencyView {
             detailLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -.smallSpacing),
 
             buttonStackView.topAnchor.constraint(equalTo: scrollView.bottomAnchor, constant: .mediumLargeSpacing),
-            buttonStackView.leadingAnchor.constraint(greaterThanOrEqualTo: leadingAnchor, constant: .mediumLargeSpacing),
+            buttonStackView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: .mediumLargeSpacing),
```

Comment:
From the design on Zeplin it was specified that it would be attached to the right side. Maybe a bit _Androidy_, but this would stretch the buttons to the entire width of the view regardless of how disproportionate the buttons will be.

Reply:
I think this looks more aligned and also the buttons aren't too close to each other. Having them the way they are today increases the likelihood of pressing the wrong button.